tests/functional/modules/test_zos_mvs_raw_func.py

```diff
@@ -55,6 +65,8 @@ def test_disposition_new(ansible_zos_module):
     try:
         hosts = ansible_zos_module
         default_data_set = get_tmp_ds_name()
+        idcams_dataset, idcams_args = get_temp_idcams_dataset(hosts)
```

Comment:
not a big deal, but perhaps there is a more descriptive name for `idcams_args`, maybe something like `idcams_listcat_cmd` or `idcams_listcat_dataset_cmd` or `listcat_idcams_dataset_cmd`?

```python
      "dd_input":{
          "dd_name":SYSIN_DD,
          "content": listcat_idcams_dataset_cmd
      }
```
Even out of context, i'd have some idea of what the "content" section is and what this test case is doing